Integrations/src/main/java/io/deephaven/integrations/python/PythonMergedListenerAdapter.java

```diff
@@ -91,6 +101,19 @@ public ArrayList<TableUpdate> currentRowsAsUpdates() {
 
     @Override
     protected void process() {
-        pyCallable.call("__call__");
+        try {
+            pyListenerCallable.call("__call__");
+        } catch (Exception e) {
+            if (!pyOnFailureCallback.isNone()) {
```

Comment:
I wonder if we should make the failure callback mandatory, and let the Python code supply a default?
```suggestion
            if (pyOnFailureCallback != null && !pyOnFailureCallback.isNone()) {
```

Reply:
I would make it mandatory in Java.